frontend/packages/yki/src/utils/table.ts

```diff
@@ -0,0 +1,19 @@
+import { t } from 'i18next';
+
+import { translateOutsideComponent } from 'configs/i18n';
+
+export class TableUtils {
+  t = translateOutsideComponent();
+
+  static getPaginationBackButtonProps() {
+    const label = t('yki.common.component.table.pagination.backButtonLabel');
+
+    return { 'aria-label': label, title: label };
+  }
+
+  static getPaginationNextButtonProps() {
+    const label = t('yki.common.component.table.pagination.nextButtonLabel');
+
+    return { 'aria-label': label, title: label };
+  }
+}
```

Comment:
Näitä käytetään nyt vain yhdestä paikasta ja nää voisi tietysti myös olla inlinenä käyttöpaikassa. Periaatteessa nää käännökset voisi korvata nyt material-UI:sta tulevat oletustekstit, mutta tuolla shared-kirjastossa ei tietystikään ole tukea käännöksille.

Reply:
Mä mielellään kyllä ottaisin nää muutokset mukaan tähän PR:ään, ihan jo siitä syystä että olisi edes jossain paikassa toteutettuna nää tarvittavat muutokset. 

En oo varma miten noiden shared-komponenttien käännökset pitäisi lopulta toteuttaa. Jokaisella ajossa olevalla sovelluksella pitäisi kuitenkin olla aina yksi i18n-instanssi / -konfiguraatio, ja vaatii ehkä hieman ajatusta jotta se saadaan fiksusti tehtyä.

Reply:
Okei.. joo, ja jos ylipäätään menee vaikeaksi / ei mielekästä yrittää käännöksiä sharedin päähän lisätä niin vastaavankaltaisen TableUtils-luokan vois kai copy pasteta myöhemmin muihin projekteihin. Ei meillä kuitenkaan ole sitten kuin 4 kopioita tällaisesta (ellei sitten osan logiikasta siirrä kuitenkin sharediin ja ota vaikka propseissa translate-funkkaria).